### PR TITLE
My jetpack connection footer logos to list

### LINKED ISSUE #35486: Jetpack Connection: list of connection services should be a list in the markup
In the "Connect Account" page, the list of services should be an actual list in the markup.

Image

Issue reported in p1HpG7-qVW-p2.

### PR DESCRIPTION
**Pull request by andrii-lysenko**
Fixes https://github.com/Automattic/jetpack/issues/35486

## Proposed changes:

Use list for the items on Jetpack Connection page footer to address accessibility issues

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Create a new JN site using this branch
* Navigate to My Jeetpack page and click 'Finish setting up Jetpack'
* Check that icons on the left side in footer look normal and are the part of the list.

### CODE REVIEW

File: projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx [hunk 1]
```diff
@@ -23,23 +23,29 @@ const ConnectionScreenFooter = () => {
...
-					className={ styles.apple }
-					alt={ __( 'Apple Logo', 'jetpack-my-jetpack' ) }
-				/>
-			</span>
+			<ul className={ styles[ 'account-images' ] }>
+				<li>
+					<img
+						src={ wordpressLogo }
+						className={ styles.wordpress }
+						alt={ __( 'WordPress Logo', 'jetpack-my-jetpack' ) }
```
Comment (monsieur-z) [inline projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx:31]:
```suggestion
						alt={ __( 'WordPress.com', 'jetpack-my-jetpack' ) }
```

File: projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx [hunk 2]
```diff
@@ -23,23 +23,29 @@ const ConnectionScreenFooter = () => {
...
+					/>
+				</li>
+				<li>
+					<img
+						src={ googleLogo }
+						className={ styles.google }
+						alt={ __( 'Google Logo', 'jetpack-my-jetpack' ) }
```
Comment (monsieur-z) [inline projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx:38]:
```suggestion
						alt={ __( 'Google', 'jetpack-my-jetpack' ) }
```

File: projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx (hunk 1 above)
Comment (monsieur-z) [inline projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx]:
```suggestion
						alt="WordPress.com"
```

File: projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx (hunk 2 above)
Comment (monsieur-z) [inline projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx]:
```suggestion
						alt="Google"
```

File: projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx [hunk 3]
```diff
@@ -23,23 +23,29 @@ const ConnectionScreenFooter = () => {
...
+					/>
+				</li>
+				<li>
+					<img
+						src={ appleLogo }
+						className={ styles.apple }
+						alt={ __( 'Apple Logo', 'jetpack-my-jetpack' ) }
```
Comment (monsieur-z) [inline projects/packages/my-jetpack/_inc/components/connection-screen/index.jsx]:
```suggestion
						alt="Apple"
```

**Review by monsieur-z — DISMISSED** [review]
Looks good!

I've proposed some changes for the `alt` attributes so that screen readers just read the name of the services. I think the experience makes more sense like that.

**Comment by andrii-lysenko** [comment]
Thanks, I agree that makes more sense. Committed suggestions.

**Review by monsieur-z — APPROVED** [review]
🚀

**Pull request merged**